nagios/tests/compose/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM jasonrivers/nagios:latest
+COPY --chown=nagios:nagios nagios4/nagios.cfg /opt/nagios/etc/nagios.cfg
+RUN  chown -R nagios:nagios /opt/nagios/var/
```

(nit) missing ending newline.
```suggestion
RUN  chown -R nagios:nagios /opt/nagios/var/

```